src/components/InfiniteScroll/index.js

```diff
@@ -49,26 +53,39 @@ export default class InfiniteScroll extends Component {
   }
 
   render() {
-    return (
-      <Grid container spacing={this.props.spacing}>
-        {this.props.data}
-        {this.props.loadingMore && <LoadingMore />}
+    const { data, spacing, loadingMore, store, width } = this.props;
+    const { createEvent, ui } = store;
+    return data.length > 0 ? (
+      <Grid container spacing={spacing}>
+        {data}
+        {loadingMore && <LoadingMore />}
       </Grid>
+    ) : (
+      <Row>
+        <Row><img src="/images/empty.svg" alt="empty placeholder" /></Row>
```

Comment:
nit: your choice, but seeing as the duplicate code in multiple places its prob best to make its own component:

**NoItemsPlaceholder**
```
const messages = defineMessages({
  // messages of the different empty placeholder messages
});

const NoItemsPlaceholder = ({ intl, myMessageId }) => (
  <div>
    <Row><img src="/images/empty.svg" alt="empty placeholder" /></Row>
    <Row>
       <Typography variant="title">
         {intl.formatMessage(message.myMessageId)}
       </Typography>
    </Row>
  </div>
);

export default (NoItemsPlaceholder);
```

Reply:
I will component it